### ISSUE TITLE
Improve error suggestion

### ISSUE THREAD

**Issue by DanielCouzens** [epimorphics/standard-reports-ui#120]
- [x] [3.3.3: Error Suggestion](https://docs.google.com/spreadsheets/d/1YS3N5ps9pJGFOxBDi9C2DwA37WQPIVax/edit?gid=1282556960#gid=1282556960&range=A12)

**Comment by DanielCouzens** [comment]
This is in review. PR https://github.com/epimorphics/standard-reports-ui/pull/125

**Comment by DanielCouzens** [comment]
This has been merged into main

**Comment by tomguilbert** [comment]
@DanielCouzens I think the requirement is for more context but this is just a wordier generic error message. I would say that for each of the 3 kinds of postcode entry you should have an error message of the form:

>Sorry, 'B170' does not appear to be a valid value for a postcode sector. A postcode sector is the first part of a UK postcode, up to and including the first digit after the space. For example "B17 0" or "TA9 3"

but with different text for each of the 3 taken from the guidance on the input form.

**Comment by DanielCouzens** [comment]
This is now in review. PR https://github.com/epimorphics/standard-reports-ui/pull/132

**Comment by DanielCouzens** [comment]
This is now merged into dev

**Comment by tomguilbert** [comment]
Looks great now thanks @DanielCouzens
